fmriprep/workflows/epi.py

```diff
@@ -738,7 +753,12 @@ def init_func_reports_wf(reportlets_dir, freesurfer, name='func_reports_wf'):
                                          ('acompcor_report', 'in_file')]),
         (inputnode, ds_tcompcor_report, [('source_file', 'source_file'),
                                          ('tcompcor_report', 'in_file')]),
-        ])
+                    ])
+    if use_aroma:
+        workflow.connect([
+            (inputnode, ds_ica_aroma_report, [('source_file', 'source_file'),
+                                              ('ica_aroma_report', 'in_file')]),
+                        ])
```

Comment:
This `if` block should stay. By not making the connection, the node won't get run.